community/kernel/pom.xml

```diff
@@ -134,7 +134,7 @@ the relevant Commercial Agreement.
         <configuration>
           <dotGitDirectory>${git.directory}</dotGitDirectory>
           <failOnNoGitDirectory>false</failOnNoGitDirectory>
-          <useNativeGit>true</useNativeGit>
+          <useNativeGit>false</useNativeGit>
```

Comment:
This is actually a very good example of a change that should have been in a separate commit since there is no way for me to understand why this change has been made.